### ISSUE TITLE
error building on ubuntu

### ISSUE THREAD

**Issue by chreman** [BGWoodward/FishAnnotator#74]
Hello,

when trying to build on ubuntu following the readme, I get following error:
```
Scanning dependencies of target image_annotator
[ 55%] Building CXX object src/image_annotator/CMakeFiles/image_annotator.dir/image_annotation.cc.o
[ 58%] Building CXX object src/image_annotator/CMakeFiles/image_annotator.dir/main.cc.o
[ 62%] Building CXX object src/image_annotator/CMakeFiles/image_annotator.dir/mainwindow.cc.o
[ 65%] Building CXX object src/video_annotator/CMakeFiles/video_annotator.dir/main.cc.o
In file included from .../projects/FishAnnotator/src/video_annotator/mainwindow.h:25:0,
                 from .../projects/FishAnnotator/src/video_annotator/main.cc:5:
.../projects/FishAnnotator/src/video_annotator/video_annotation.h:431:21: error: non-type partial specialization ‘getRequired<FieldType>’ is not allowed
     FieldType &value) {
```
Please let me know if any further info is required,
Best regards Chris

**Comment by BGWoodward** [comment]
We are looking into this now. Can you confirm what version of ubuntu and what version of gcc you are using? This is possibly related to C++11, which we use in this project.

**Comment by chreman** [comment]
gcc --version gives that:
```
gcc --version
gcc (Ubuntu 5.4.0-6ubuntu1~16.04.4) 5.4.0 20160609
```

**Comment by BGWoodward** [comment]
Thank you. I was able to reproduce the error. For now if you revert to commit 

5a2ab913e6c901a280fad9d92cf4c270e95e931e

you should be able to build but will only be missing the ability to seek to a specific frame, and in order to view multiple videos or annotation files, you will need to restart the application. We anticipate pushing a fix for this today or tomorrow.

Thanks,

-Ben

**Comment by chreman** [comment]
Will do. Thank you for the quick reaction :)

**Comment by jrtcppv** [comment]
Closed by 415a0883004a3171c36a6777b8b5673554079715.  You should be good to go back to the latest on master.
